### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by VaheStepanyan100**
- Modify the Awesome books application to have:

  - A Navigation bar.
  - Three content sections:
  - Books list.
  - Add book form.
  - Contact info.
- Use this wireframe as a guide:
- Make sure that when you click on any of the links, the main area changes to display only one of the three content sections.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,58 +1,126 @@
...
+//   const title = document.querySelector('.title');
+//   const author = document.querySelector('.author');
+//   e.preventDefault();
+//   addNewBook(title.value, author.value);
+// });
+
+// removeBook();
+
+// displayBooks();
+
```
Comment (emmiiorji) [inline script.js:59]: Kindly remove commented blocks of code in order to clean up the code base.

File: script.js [hunk 2]
```diff
@@ -1,58 +1,126 @@
...
+  }
+
+  updatelocalStorage() {
+    localStorage.setItem('Added Books', JSON.stringify(this.books));
+  }
 
-function updatelocalStorage() {
-  localStorage.setItem('Added Books', JSON.stringify(storedBooksList));
-}
+  /* eslint-disable class-methods-use-this */
```
Comment (emmiiorji) [inline script.js:73]: As much as possible, kindly avoid disabling linter checks in line. Linters help us to maintain a uniform code base and adhere to coding standards.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @VaheStepanyan100, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

<div align='center'>

</div>

You have submitted a pair programming activity without participation of your other team member(s). Kindly ensure that every member of the team participates in the project evidenced by their commits.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by AlphaNtihinduka — COMMENTED** [review]
Hello Team
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request ❌ 

- [x] You have submitted a pair programming activity whose participation of your other team member is not valid. Kindly ensure that every member of the team participates in the project with real changes other than deleting the commented code evidenced by their commits.
- [x] [OPTIONAL] You may also need to update the Readme authors so that the contributor is also part of your readme.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: script.js [hunk 3]
```diff
@@ -2,57 +2,88 @@ if (localStorage.getItem('Added Books') == null) {
...
-function removeBook(id) {
-  storedBooksList = storedBooksList.filter((el) => el.id !== id);
-  updatelocalStorage();
-  displayBooks();
+  removeBook(id) {
+    this.books = this.books.filter((el) => el.id !== id);
+    this.updatelocalStorage();
+    this.displayBooks();
+  }
+
```
Comment (Ismailco) [inline script.js:52]: - [ ] Your code should have a class `methods` to `add` and `remove` books, kindly consider creating a class with that name (methods) and include the methods responsible for adding and removing the books to it

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @VaheStepanyan100, @leloufadel,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ UI like the wireframe
✔️ No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 3 above)
Comment (VaheStepanyan100) [reply script.js:52]: I have already created class methods for adding and removing. This part of the code is already updated

**Review by V-Blaze — APPROVED** [review]
Hi @leloufadel, @VaheStepanyan100  

# Approved 🍏 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
green-lantern-ok

`My apologies for the error from the previous reviewer, The code he highlighted really doesn't exist in your codebase :+1: :smile: `

## Highlights
- Used JavaScript classes✔✔
- addBook and removeBook works properly :heavy_check_mark: :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — COMMENTED** [review]
Hi @VaheStepanyan100,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

I noticed that the pull request for the new milestone was not created as a new request, but instead, the description of the existing pull request was edited. It's important to create a new pull request for each new milestone to ensure that the changes made are easily tracked and reviewed.

Now, please create a new pull request for each new milestone, even if it's related to a previous pull request. This will make it easier for us to keep track of changes and ensure that everything is properly reviewed.

Thank you for your attention to this matter, and please let me know if you have any questions or concerns.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**